phys/module_pbl_driver.F

```diff
@@ -537,7 +540,10 @@ SUBROUTINE pbl_driver(                                          &
                                                          RVBLTEN, &
                                                         RTHBLTEN, &
                                            EXCH_H,EXCH_M,TKE_PBL, &
-                                                        RTHRATEN  ! for GBM PBL scheme
+                                                DISS_PBL,TPE_PBL, &
+                                        TKE_ADV,DISS_ADV,TPE_ADV, &
+                                                 PR_PBL,RTHRATEN  ! for GBM PBL scheme
```

Comment:
@andreazonato If you add PR_PBL on this line, then the comment at the end isn't correct any more. Can remove the comment.

Reply:
Yes, all modifications make sense. Shall I do it or you already made them?

Reply:
@andreazonato you can do it